### PR TITLE
MusicStore sample unify, add .NET 6

### PR DESCRIPTION
**Pull request by korser1**
+use props file for MusicStore sample
+add .NET 6
*update Steeltoe version
*update Azure libs version
*update docker files to .NET 6
*fix solutions links

### CODE REVIEW

File: MusicStore/src/MusicStoreUI/MusicStoreUI.csproj [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFrameworks>netcoreapp3.1;net6.0</TargetFrameworks>
```
Comment (TimHess) [inline MusicStore/src/MusicStoreUI/MusicStoreUI.csproj:3]:
```suggestion
    <TargetFrameworks>net6.0</TargetFrameworks>
```

File: MusicStore/src/OrderService/OrderService.csproj [hunk 2]
```diff
@@ -1,37 +1,36 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFrameworks>netcoreapp3.1;net6.0</TargetFrameworks>
```
Comment (TimHess) [inline MusicStore/src/OrderService/OrderService.csproj:3]:
```suggestion
    <TargetFrameworks>net6.0</TargetFrameworks>
```

**Review by TimHess — COMMENTED** [review]
Thanks @korser1 for working on this. Something is off though, the solution won't built in Visual Studio for me, with this error for each project:
`Samples\MusicStore\src\MusicStoreService\MusicStoreService.csproj : error MSB4057: The target "DockerResolveAppType" does not exist in the project`

Some quick research indicates that the tooling doesn't like multi-targeting, and I can confirm that message goes away if I use <TargetFramework> singular instead of plural. But then the version variables in the props files don't work anymore... I think it might make sense to put the .NET 6 package versions into the static block of versions in the .props files, and have them only overridden when the target is .NET Core 3.1, but I won't have time to test that today

**Comment by korser1** [comment]
Updated

**Review by TimHess — APPROVED** [review]
LGTM, Thank you!

**Pull request merged**